### ISSUE TITLE
unicode characters encoding issue?

### ISSUE THREAD

**Issue by blizzz** [domschrei/krunner-symbols#11]
I am on a Arch-based distro, got symbols from AUR and enabled unicode. However, most characters are just show with the placeholder, also after copying:

![screenshot_20180118_122837](https://user-images.githubusercontent.com/2184312/35096086-1f6d3a4e-fc4b-11e7-9f09-ca095479a5d5.png)

**Comment by domschrei** [comment]
I had this issue on my installation as well, and I thought that it would just be some missing font packages. But now I actually did some research, looked into the incorrectly displayed symbols in the lookup table and figured that it concerned all symbols whose hexadecimal encoding has length 5 (the most common symbols all have length 4). I fixed the issue now by adding a special treatment for those symbols, explicitly decoding them into the usual unicode format. 

So now it should work, at least it does for me: 👍. Thank you for the heads up! I wouldn't have looked into this if you didn't submit the issue.

EDIT: To clarify, I do still get some crossed out boxes when searching for some emojis, e.g. "dog face", but these are actually due to the font I am using for krunner: when pressing enter and pasting it in the browser, I get "🐶" (should display as dog face).
